scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -262,12 +263,12 @@ def run_cmake(self, args=""):
         cmake_opts = ['-DBOARD={}'.format(self.platform.name)]
         cmake_args.extend(cmake_opts)
 
-
-        logger.debug("Calling cmake with arguments: {}".format(cmake_args))
         cmake = shutil.which('cmake')
         cmd = [cmake] + cmake_args
         kwargs = dict()
 
+        logger.debug("Calling cmake: %s", shlex.join(cmd))
```

Comment:
Looks like `shlex.join` is a new function in Python 3.8.

It does not really matter anyway since we are going to be updating the minimum required Python version to 3.8 (though, strictly speaking this PR should be merged after that is done).

Reply:
>  this PR should be merged after that is done

Thank you for catching that; updated accordingly.